runtime/basilisk/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "basilisk-runtime"
-version = "38.0.0"
+version = "39.0.0"
```

Comment:
@lumir-mrkva  well, runtime change again. 

in this case - is it really  needed here ?  there is only one change and it only affects benchmarking.

Reply:
critical bug in xyk was fixed